Doc-2455 support datetime parameter type when running interpret querythrough endpoint api [4.2.0-dev] (#738)
tigergraph/server-docs: modules/release-notes/pages/index.adoc

* [4.1.1] Added token function **xref:4.1@gsql-ref:ddl-and-loading:creating-a-loading-job.adoc#_token_functions[gsql_current_datetime()]** for loading jobs.

> * Added support for using the `datetime` function as a parameter value within INTERPRET QUERY. This enhancement allows users to assign date strings and leverage string functions when passing parameters in interpreted queries, improving flexibility in query execution.

Reviewer: ```suggestion
* Added support for using the `datetime` function in a parameter when running INTERPRET QUERY.
```
Programmers know the value of being able to use functions. We don't need to explain the benefit.
